### PR TITLE
Remove Waterpark Extras

### PR DESCRIPTION
**Pull request by KTong821**
This removes the entirety of the clients folder, as well as the population scripts used to create new clubs and events. These were carried over during the Waterpark codebase clone, and should be removed now that they are no longer necessary.

### CODE REVIEW

File: README.md
Comment (xujustinj) [inline README.md:1]: Copied over from #14 

TODO: update this to relate to auth. Not sure if that's something we want to add to this PR.

**Review by xujustinj — APPROVED** [review]

**Review by xujustinj — APPROVED** [review]

**Comment by xujustinj** [comment]
I would wait until after #18 is merged though, the merge conflicts will probably be easier to handle in this branch

**Comment by ShreyasPrasad** [comment]
Not sure if I specified this earlier, (really sorry if I didn't) but the clients folder is actually required given that the auth project will have it's own FE components (mostly just login/creation flows, password reset etc). @KTong821 @xujustinj  Will make a quick PR for the stuff included in this PR that should still be removed regardless.

The new PR is #20, keeps the client folder intact.

**Pull request closed without merging**
